packages/phone-number-privacy/common/src/interfaces/requests.ts

```diff
@@ -28,24 +28,25 @@ export enum AuthenticationMethod {
   ENCRYPTION_KEY = 'encryption_key',
 }
 
-// (https://github.com/celo-org/celo-monorepo/issues/9804)
-// TODO(2.0.0, refactor) improve pnp request/response types for new vs. legacy endpoints
-// TODO(2.0.0, refactor): investigate alias in sdk/identity
-export interface GetBlindedMessageSigRequest {
+export interface SignMessageRequest {
   /** Celo account address. Query is charged against this account's quota. */
   account: string
   /** Query message. A blinded elliptic curve point encoded in base64. */
   blindedQueryPhoneNumber: string
   /** Authentication method to use for verifying the signature in the Authorization header */
   authenticationMethod?: string
-  /** Optional on-chain identifier. Unlocks additional quota if the account is verified as an owner of the identifier. */
-  hashedPhoneNumber?: string
   /** Client-specified session ID for the request. */
   sessionID?: string
   /** Client-specified version string */
   version?: string
 }
-export declare type SignMessageRequest = GetBlindedMessageSigRequest
+export interface LegacySignMessageRequest extends SignMessageRequest {
+  /** Optional on-chain identifier. Unlocks additional quota if the account is verified as an owner of the identifier. */
+  hashedPhoneNumber?: string
+}
+
+// Backwards compatibility
```

Comment:
note: given that this is a 2.0.0 release, happy to just delete `GetBlindedMessageSigRequest`!